playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -62,6 +64,10 @@
   roles:
   - openshift_facts
   post_tasks:
+
+  - include: "{{ openshift_upgrade_pre_master_hook }}"
```

Comment:
:+1: to the simplicity of this.

I'm a paranoid guy when I run things. It may be nice to have an output statement letting the user know t heir hook is being executed. Obviously not a blocker.